geex-core/src/main/java/nl/tudelft/context/controller/MainController.java

```diff
@@ -144,6 +144,10 @@ public final void setBaseView(final ViewController viewController) {
      */
     public void setView(final ViewController on, final ViewController viewController) {
 
+        if (overlay.isVisible()) {
+            menuController.getToggleOverlay().fire();
+        }
+
```

Comment:
I got some intense errors when I used that, but didn't really think much about it because this worked anyway.